doc/RULES

```diff
@@ -28,11 +28,19 @@ natively.  Those additional commands may also be useful on their own.
 -8	reject this rule unless current hash type uses 8-bit characters
 -s	reject this rule unless some password hashes were split at loading
 -p	reject this rule unless word pair commands are currently allowed
+-R	reject this rule unless stacked rules will also be applied later
+-S	reject this rule if stacked rules will also be applied later
 -u	reject this rule if an internal codepage is configured (doc/ENCODINGS)
 -U	reject this rule unless an internal codepage is configured
 ->N	reject this rule unless length N or longer is supported
 -<N	reject this rule unless length N or shorter is supported (--min-length)
 
+Note that -R will only accept a rule when both --rules (or --single) and
+--rules-stack are used at the same time (rules stacked on rules) and even then
+it will only accept lines from --rules, not from --stacked-rules.  If you run
+only --rules OR --rules-stack, -S will accept and -R will reject.  So if/when
+deciding which of those two flags to use, -S is normally preferable.
```

Comment:
The options do the opposite things and are presumably to be used in different cases, so I find it weird to say that one is "preferable" over the other.

Reply:
> I don't really mind having this, although I also don't see much need for it. Please provide a valid use case for it before merging, or else please revise to match a use case.

The IRL case that made me do this was actually just a bunch of similar appends (although more of them and longer) and the unwanted/useless words quickly grew to a large number.

> With this, I don't have a valid use case for `-S` at all. So maybe instead of my suggested `-D` above, we should redefine the meaning of `-R` and `-S` to be "reject this rule except when stacked along with a different rule" and "reject this rule when stacked along with itself", respectively. Then on `[PI]` we only need `-S` and on `:` we only possibly need `-R`. I think I like this better.

I like your ideas better too. I'll experiment with that!